### PR TITLE
qa/tests: Added mimic runs, removed large suites (rados, rbd, etc) ru…

### PR DESCRIPTION
**Pull request by yuriw**
…ns on luminous

FYI  - we will need this ? https://tracker.ceph.com/issues/24021

Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

### CODE REVIEW

File: qa/crontab/teuthology-cronjobs [hunk 1]
```diff
@@ -150,6 +126,59 @@ CEPH_QA_EMAIL="ceph-qa@ceph.com"
...
+#35 05 * * 1,3,5 CEPH_BRANCH=mimic; MACHINE_NAME=smithi;/home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 7 -m $MACHINE_NAME -s samba -e $CEPH_QA_EMAIL
+55 05 * * 1,3,5  CEPH_BRANCH=mimic; MACHINE_NAME=mira; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 7 -m $MACHINE_NAME -s ceph-deploy -k distro -e $CEPH_QA_EMAIL
+10 05 * * 1,3,5  CEPH_BRANCH=mimic; MACHINE_NAME=ovh;  /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 7 -m $MACHINE_NAME -s ceph-disk -k distro -e $CEPH_QA_EMAIL
+15 05 * * 1,3,5  CEPH_BRANCH=; MACHINE_NAME=ovh;  /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 7 -m $MACHINE_NAME -s ceph-ansible -k distro -e $CEPH_QA_EMAIL
+
+
+25 02 * * * CEPH_BRANCH=mimic; MACHINE_NAME=ovh; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -k distro -n 7 -m $MACHINE_NAME -s upgrade/luminous-x -e $CEPH_QA_EMAIL --suite-branch $CEPH_BRANCH -p 90 --filter ubuntu_latest,centos
+
+## upgrades suites for on mimic
+## !!!! three suites below MUST use --suite-branch hammer OR jewel
```
Comment (liewegas) [inline qa/crontab/teuthology-cronjobs]: ?

**Comment by liewegas** [comment]
looks ok to me!

File: qa/crontab/teuthology-cronjobs (hunk 1 above)
Comment (liewegas) [reply qa/crontab/teuthology-cronjobs]: is the comment accurate?  should we add luminous now?  also doesn't seem to include kraken?

Comment (yuriw) [reply qa/crontab/teuthology-cronjobs]: see now, fixed comments and `client-upgrade-luminous suite` is being worked on, see  https://tracker.ceph.com/issues/24021 and https://github.com/ceph/ceph/pull/21828

**Pull request merged**